api/RTCPeerConnection.json

```diff
@@ -1726,9 +1726,9 @@
             "deprecated": false
           }
         },
-        "MediaStreamTrack_argument": {
+        "MediaStreamTrack_parameter": {
```

Comment:
I still don't like this name but don't know what it should be per https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#parameters-and-parameter-object-features

*All* arguments are unnamed, so read pedantically this should just be `first_parameter`. But if not for that, maybe `selector_parameter` would be better, since that's what the parameter is called by convention.

Reply:
Arguments are unnamed in the sense that the names cannot be observed by web developers. If the arguments were renamed in specs and implementations to random strings, everything would continue to work the same for web developers.

Dictionary members however do have names, which have to appear at the call sites. Compare:
```js
let ev = new Event("hey", {bubbles:true}); // bubbles is a name!
document.dispatchEvent(ev); // the first argument's name "event" appears nowhere
```

Reply:
Ah, OK. I understand now. I guess my take here is that we should always go for a name, if at all possible, in the description. Names may not be strictly observable, but they're not completely hidden from developers either (e.g., when I type `new Event(`, VS Code shows me a tooltip for the `type` argument, presumably derived from the IDL).

If you want to name the feature keys by the ordinal though, I might be convinced of that. We could start a new discussion on the guideline, if you like. I'm willing to entertain some level of pedantry. 😄

Reply:
No, I also think we should use the spec names for arguments, just noting that a pedantic reading of the guideline didn't work :)

Reply:
OK, so want to rename this to `selector_parameter` instead and revise the description to match?

Reply:
Yes!